frontend/src/views/ExtractView.vue

```diff
@@ -82,6 +82,13 @@ const extractFrame = async () => {
     selectedFrame.value = framesList.value.indexOf(data.value[0]);
   }
 };
+
+// Change player to frame of clicked of element
```

Comment:
For consistency can you use lowercase for 'change', also I think it's meant to be 'on' instead of 'of'.
```suggestion
// change the player to the frame extracted from the images name
```